packages/letters-of-credit/src/components/Pages/MatiasPage/MatiasPage.js

```diff
@@ -4,6 +4,7 @@ import { Redirect } from 'react-router-dom';
 import axios from 'axios';
```

Comment:
Same for Matías' page as for Ella's page, still missing a few of the things in the designs

![3 - matiashome](https://user-images.githubusercontent.com/35696678/39422072-d361ba12-4c63-11e8-99e7-fd252c8087a1.png)